aula-css/exercicios/exercicio 1 css/detalhes-1.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="details.css">
+</head>
+<header>
+    <div class="titulo">
+        
+        <div>
+            <p><h1>Leila´s Photo Grid</h1></p>
+            
+        </div>
+        <br>
+        <div class="botao">
+        
+            <a href="index.html"><button> < Voltar </button> </a>
+        </div>
+    </div>
+    
+
+</header>
+        
+
+
+    <section class="container">
+
+
+        <div class="imagens">
+          
+            <div class="fotos">
+                <img src="imagem/gato na arvore2.jpg" alt="Gato na árvore">
+                
+            </div>
+            
+        </div>
+        <div class="perfil">
+            
+            <div class="descricao">
+                <img src="imagem/cat.jpg" alt="foto de perfil é do gato">
+                <p><h1>leilagram</h1></p>
```

Comment:
```suggestion
                <h1>leilagram</h1>
```

Mesma coisa que falei mais acima sobre títulos e parágrafos :)